### PR TITLE
fix styles

### PR DESCRIPTION
**Pull request by britaumich**

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <%= render './layouts/headness.html.haml' %>
 <% if user_signed_in? && current_user.admin %>
   <% if api_status == 'failed' %>
-    <div class="grid w-full bg-yellow-550 text-sm place-content-center py-1">
-      The Classrooms Update Script failed to run today. Please report the issue.
+    <div class="flex flex-row justify-center items-center bg-red-800 text-red-100 text-sm place-content-center py-1">
```
Comment (anantaUM) [inline app/views/layouts/_header.html.erb:4]: Do we need all three following classes  here ?

- justify-center
- items-center
- place-content-center

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+      The Classrooms Update Script failed to run today.
+      <div>
+        <a id="myCustomTrigger" class="text-sm hover:underline px-1" href="#" data-email="<%= session[:user_email]%>">Report an Issue</a>
```
Comment (anantaUM) [inline app/views/layouts/_header.html.erb:7]: I think that "**Report an Issue**" text should always be underlined and not just on hover. May be a dotted line by default which turns to solid line on hover. This way the user knows that the text is a link and is clickable.

**Review by anantaUM — COMMENTED** [review]
Added review comments.

**Comment by anantaUM** [comment]
Also, why is there a button on line 14 of this file ?

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (rsmoke) [reply app/views/layouts/_header.html.erb:4]: I agree that if you place a div around the actual message block and add a centering you can do it with fewer classes.

File: app/views/layouts/_header.html.erb (hunk 2 above)
Comment (rsmoke) [reply app/views/layouts/_header.html.erb:7]: Solid underline always worked make this like the footer. Let's go that route

**Review by rsmoke — CHANGES_REQUESTED** [review]
Please set link to feedback as solid underline.
Manage placement of message with a div and 
flex; align-items: center

Nice quick fix otherwise

**Comment by britaumich** [comment]
I edited the code to make the styles more simple and clean. 
[Report an Issue] link looks the same way as it looks in the footer.
The button on line 14 (now it's 11-13) is needed to show/hide the error message.

To test the functionality, run the following commands in the rails console (check the browser after every command)

- ApiUpdateLog.create(result: "No access token", status: "error")
   in the browser click on the message "No access token"

- ApiUpdateLog.last.update(status: "success")
- ApiUpdateLog.last.destroy

**Review by rsmoke — APPROVED** [review]

**Pull request merged**